cardano-config/src/Cardano/Config/Logging.hs

```diff
@@ -255,7 +255,7 @@ loggingCardanoFeatureInit disabled' conf = do
 
 startCapturingMetrics :: Trace IO Text -> IO ()
 startCapturingMetrics trace0 = do
-  let trace = appendName "node-metrics" trace0
+  let trace = appendName "node.metrics" trace0
```

Comment:
these should not be named the same as other metrics. we want to keep them in a separate name context so they can be routed differnt. please, don't change the name here.

Reply:
Done.